### PR TITLE
Notify admins what rank they are upon joining

### PR DESCRIPTION
**Pull request by alau740**
This adds a message to the welcome notification telling admins what rank they are (and almost every other admin system i'm aware of on roblox already do this)

It allows admins to understand what permission level they have in a game quicker (instead of having to run `:admins` after they join)

Here is an example for a user with moderators permissions:
Screenshot 2023-12-14 204300

It also works with custom ranks; an example is shown here with the 'Helpers' rank, admin level 50:
Screenshot 2023-12-14 205128

### CODE REVIEW

**Review by WalkerOfBacon — APPROVED** [review]
someone was bound to do this

and that someone is you

**Comment by AlexanderDarmody** [comment]
Can you remove the "s" at the end of the default ranks so it doesn't look dumb

**Review by bigbenster702 — APPROVED** [review]
:D

File: MainModule/Server/Core/Process.lua [hunk 1]
```diff
@@ -950,7 +950,7 @@ return function(Vargs, GetEnv)
 					local newVer = (level > 300) and tonumber(string.match(server.Changelog[1], "Version: (.*)"))
 
 					if Settings.Notification then
-						Functions.Notification("Welcome.", "Click here for commands.", {p}, 15, "MatIcon://Verified user", Core.Bytecode(`client.Remote.Send('ProcessCommand','{Settings.Prefix}cmds')`))
+						Functions.Notification("Welcome.", "Your rank is ".. rank ..". Click here for commands.", {p}, 15, "MatIcon://Verified user", Core.Bytecode(`client.Remote.Send('ProcessCommand','{Settings.Prefix}cmds')`))
```
Comment (SuperCater) [inline MainModule/Server/Core/Process.lua:953]: Maybe show rank level as well?
```suggestion
						Functions.Notification("Welcome.", `Your rank is {rank} ({level}). Click here for commands.`, {p}, 15, "MatIcon://Verified user", Core.Bytecode(`client.Remote.Send('ProcessCommand','{Settings.Prefix}cmds')`))
```

Comment (P3tray) [reply MainModule/Server/Core/Process.lua:953]: overkill

**Review by P3tray — CHANGES_REQUESTED** [review]
Do you know who else has dementia?

**Review by ccuser44 — APPROVED** [review]

**Comment by P3tray** [comment]
they merged the dementia command

**Comment by Dimenpsyonal** [comment]
Do you know who else has dementia?

**Comment by P3tray** [comment]
they merged the dementia command

**Comment by P3tray** [comment]
they merged the dementia command

**Comment by Dimenpsyonal** [comment]
Do you know who else has dementia?

**Comment by P3tray** [comment]
they merged the dementia command

**Pull request merged**